dev_mode/webpack.config.js

```diff
@@ -160,7 +160,7 @@ module.exports = {
     fs: 'empty'
   },
   bail: true,
-  devtool: 'cheap-source-map',
+  devtool: 'cheap-module-eval-sourcemap',
```

Comment:
As noted in the comments, can we change this back to `cheap-source-map` (or even `source-map` - it doesn't seem to take that much longer for incremental compilation for me) - the eval version makes it really tedious to debug without using the source map.